app/components/windows/EditStreamInfo.vue.ts

```diff
@@ -192,4 +223,16 @@ export default class EditStreamInfo extends Vue {
     return this.streamInfoService.state.error;
   }
 
+  get profiles() {
+    const multiselectArray: IMultiSelectProfiles[] = [];
+    const profiles = this.videoEncodingOptimizationService.getGameProfiles(this.gameModel.value);
+    profiles.forEach(profile => {
+      multiselectArray.push({
+        value: profile,
+        description: profile.profile.description,
+      });
+    });
+    return multiselectArray;
+  }
+
```

Comment:
Can you download the "Prettier - Code Formatter" extension for Visual Studio Code?  If you press Ctrl+Shift+P it will bring up the command pallette and you can search for "Format Document".  This will automatically format any `.ts` (or `.vue.ts`) files for you.  Unfortunately it will not work on `.vue` files.

Reply:
Done.